src/admin/redux/modules/letters.js

```diff
@@ -0,0 +1,57 @@
+const { createAction } = require('redux-actions')
+const { get_body } = require('app/http')
+const { merge, compose, objOf, map, props, zipWith, pick } = require('ramda')
+const formatDate = require('app/format_date')
+
+const SEND_NEWSLETTER_POST =
+  'SEND_NEWSLETTER_POST'
+const SEND_SUB_REMINDER_POST =
+  'SEND_SUB_REMINDER_POST'
+
+const initialState = {
+  post_members: [],
+  sub_reminders: []
+}
+
+const reducer = (state = initialState, { type, payload }) => {
+  switch (type) {
+  case SEND_NEWSLETTER_POST:
+    return { ...state, post_members: payload.results }
+  case SEND_SUB_REMINDER_POST:
+    const idObj = map(pick([ 'id' ]), payload.results)
```

Comment:
So should we leave initial state as an array? 

We changed to an object yesterday because react components were trying to render newsletters when we clicked on get post members and vice versa.